packages/sdk-cli/babel.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { presets: ['@babel/preset-env'] };
```

Comment:
Is this necessary? Would think it'd use the file in the root. I don't see it in my example here: https://github.com/Fitbit/developer-bridge/commit/22f93b1e8f61cdcd13c05c7ac9a5d40c32dbb519